recipes/qpdf/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,27 @@
+#include <qpdf/DLL.h>
```

Comment:
I think you meant to delete this file.

Reply:
uilianries requested to add the test_v1_package scenario to run also the "old" conan package world. I do not like the code-duplication here for the source file "test_package.cpp". What do you think would be the best solution here?

Reply:
ahh now i understand. Thanks for pointing out. I remove the unnecessary file here